godtools/Views/Cells/HomeToolTableViewCell.swift

```diff
@@ -76,7 +76,7 @@ class HomeToolTableViewCell: UITableViewCell {
         titleLabel.isEnabled = isAvailableInPrimaryLanguage
         titleLabel.text = resource.localizedName(language: primaryLanguage)
         
-        languageLabel.text = resource.isAvailableInLanguage(parallelLanguage) ? parallelLanguage!.localizedName() : nil
+        languageLabel.text = resource.isAvailableInLanguage(parallelLanguage) ? parallelLanguage!.localizedName : nil
```

Comment:
I mean, instead of that line have something like languageLabel.text = `getParallelLanguageName()` which will have all of that, and will return nil if doesn't exist. This is a common thing in the code of all app, don't know what is best